### PR TITLE
Read info doors

### PR DESCRIPTION
**Pull request by BogdanRandstad**
Added features for reading doors.
Implemented a specific class.
Implemented the respective api route and update html syntax and javascript for adding read from doors option

### CODE REVIEW

**Review by CatalinGP — APPROVED** [review]

**Review by asavu91 — APPROVED** [review]

File: src/rest_api/actions/read_info_action.py [hunk 1]
```diff
@@ -202,6 +214,49 @@ def read_from_engine(self):
         except CustomError as e:
             self.bus.shutdown()
             return e.message
+    
+    def read_from_doors(self):
+        id_door = self.id_ecu[1]
+        id = self.my_id * 0x100 + id_door
+        log_info_message(logger, "Changing session to default")
+        self.generate.session_control(id, 0x01)
+        self._passive_response(SESSION_CONTROL, "Error changing session control")
```
Comment (RubenNatanael) [inline src/rest_api/actions/read_info_action.py:223]: Is better if this two lines are added in to the try block, if an exception is throw from the response method.

File: src/rest_api/actions/read_info_action.py [hunk 2]
```diff
@@ -202,6 +214,49 @@ def read_from_engine(self):
...
+
+
+            #dummy door values param
+            DOOR=0x2001
+            DOOR_SERIALNUMBER=0x2002
+            LIGHTER_VOLTAGE=0x210D
+            LIGHT_STATE=0x221E
+            BELT_STATE=0x21C1
+            WINDOWS_CLOSED=0x230F
+
```
Comment (RubenNatanael) [inline src/rest_api/actions/read_info_action.py:241]: The identifiers must be put in the config file. Don't use dummy identifiers, set specific ones with your colleagues.

**Review by RubenNatanael — CHANGES_REQUESTED** [review]

File: src/rest_api/actions/read_info_action.py (hunk 2 above)
Comment (BogdanRandstad) [reply src/rest_api/actions/read_info_action.py:241]: The identifiers for doors don't exist on UDS side. Temporary will use dummy values.

Comment (RubenNatanael) [reply src/rest_api/actions/read_info_action.py:241]: You can set specific ones like in the battery module

**Pull request closed without merging**